### PR TITLE
Add jwt claim based access validation policy docs

### PR DESCRIPTION
**Pull request by Akila94**
## Purpose

This PR contains the doc changes for the JWT claim based access policy.

Related dev PR: https://github.com/wso2/carbon-apimgt/pull/12247
Policy PR: https://github.com/wso2/product-apim/pull/13361

### CODE REVIEW

File: en/docs/design/api-policies/regular-gateway-policies/jwt-claim-based-access-validator.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+1.  Select any API and from the Left Menu, go to **API Configurations** --> **Policies**.
+2.  Under the **Policy List** that appear on the right side of the screen, look for the **JWT claim based access validator** policy from the `Request` tab.
+
+    [![JWT claim based access validator]({{base_path}}/assets/img/design/api-policies/jwt-claim-based-access-validation-policy.png)]({{base_path}}/assets/img/design/api-policies/jwt-claim-based-access-validation-policy.png)
+
+3.  Drag and drop the **JWT claim based access validator** policy from the policy list to the request flow of any desired API operation. In the below screenshot, the policy was dropped to the `/order POST` operation.
+
+    Configure the custom claim name and value. If required, configure a regex so the claim value will be validated using it.
+
+    In the policy configuring panel that appear from the right, select `Apply to all resources` option if you wish to attach the disable chunking policy to each and every resource of the current API. If you only wish to attach the policy to a particular API operation, leave the checkbox as it is. Then, click on **Save** button.
```
Comment (tharikaGitHub) [inline en/docs/design/api-policies/regular-gateway-policies/jwt-claim-based-access-validator.md:18]:
```suggestion
    In the policy configuring panel that appears from the right, select `Apply to all resources` option if you wish to attach the disable chunking policy to each and every resource of the current API. If you only wish to attach the policy to a particular API operation, leave the checkbox as it is. Then, click on **Save** button.
```

Comment (tharikaGitHub) [inline en/docs/design/api-policies/regular-gateway-policies/jwt-claim-based-access-validator.md:18]: What is referred to as the `disable chunking policy` in this paragraph?

File: en/docs/design/api-policies/regular-gateway-policies/jwt-claim-based-access-validator.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    [![JWT claim based access validator]({{base_path}}/assets/img/design/api-policies/configure-jwt-claim-based-access-validation-policy.png)]({{base_path}}/assets/img/design/api-policies/configure-jwt-claim-based-access-validation-policy.png)
+
+4.  Finally, scroll down and click on the **Save** button in order to apply the attached policies to the API.
+
+    [![JWT claim based access validator]({{base_path}}/assets/img/design/api-policies/save-attached-policies.png)]({{base_path}}/assets/img/design/api-policies/save-attached-policies.png)
+
+Once the API is deployed and published, every request goes to this API resource will be validated against the custom claim comes in the access token before sending it to the backend.
```
Comment (tharikaGitHub) [inline en/docs/design/api-policies/regular-gateway-policies/jwt-claim-based-access-validator.md:26]:
```suggestion
Once the API is deployed and published, every request that goes to this API resource will be validated against the custom claim that comes in the access token before sending it to the backend.
```

**Comment by Akila94** [comment]
Fixed the suggestions.

**Review by tharikaGitHub — APPROVED** [review]

**Pull request merged**